01_01_Basic_Rules.txt

```diff
@@ -0,0 +1,1311 @@
+
+Base Stats:
+	These decide what raw characteristics make up your character, and are 
+factored into most other decisions you make both in game and for the rest of 
+character creation. Choosing your stats is a very good way to start making a 
+character.
+
+Species/Race:
+	Is your character a robot? What about a human? Or maybe the catlike aliens 
+known as Tundari? Different species and races get small bonuses to different 
+aspects of combat and are liked or disliked by different people in the galaxy 
+accordingly.
+
+Class: 
+	Think of this like your job. Are you a sniper, waiting for the perfect shot? 
+Or are you a pilot who can thread a needle with your starfighter? Or maybe 
+you're a veteran soldier, who's seen it all. Classes define what armor and 
+weapons your character can use in combat, as well as certain special abilities 
+which might help solve puzzles or get you better discounts at stores. Your class 
+also determines how much money you start with to buy equipment and how many 
+Nanites you can use at once.
+
+Nanites:
+	The evolution of technology brought about the ability to design and control 
+microscopic machines. Such a powerful advancement assists in all forms of life 
+in the Compound X universe. Nanites are used to activate powerful abilities such 
+as sprinting extremely fast, aiming incredibly well, infusing bullets with 
+electrical energy, and more.
+
+Skills: 
+	Each character learns things while going through life. Maybe your time in 
+the military taught you how to navigate using the stars or maybe your time 
+working at a mechanic's shop made you good at repairing things? Or maybe you're 
+one of those people who is just naturally good at hacking computers. Picking 
+these numbers will represent that in the game world.
+
+Equipment:
+	Your equipment is made up of your character's weapon, armor and 
+miscellaneous items. You'll get money to buy them with at the start of the game 
+and you'll earn more money as you go about your adventures.
+
+Feats: 
+	Your character will gain special perks as you level up. This could range 
+from reloading really fast, to having more health to being able to do a fast 
+combat roll under fire. Tackle these when your character advances to level 3.
+
+==============================
+Base Stats
+==============================
+
+	Each player has what are called Base Stats. An average person has a '5' for
+each stat, but an average player has a '6'. Base Stats are split into Primary
+and Secondary stats. Primary stats are determined by your Base Stat points.
+Secondary stats are derived from primary stats. The largest primary stat
+associated with a given secondary stat determines the secondary stat's value; 
+However, if you have a primary stat that is associated with more than one 
+secondary stat, you may have to choose which secondary stat to contribute to.
+These stats are as follows (secondary stat associations are indented):
+<<<<<<< HEAD

```

Comment:
Head, shoulders, knees, and toes. Lol (line 179)

Reply:
I'd call these blockers.